packages/ignite-basic-generators/index.js

```diff
@@ -1,9 +1,27 @@
+const screenExamples = [
+  {
+    title: 'Row Example', 
+    screen: 'examples/RowExample.js', 
+    ancillary: ['examples/Styles/RowExampleStyle.js']
```

Comment:
I find the word `ancillary` confusing.  Your vocabulary is much richer than mine though.  I'd have chosen `files` and would have explained in the docs these are extra goodies.